src/main/java/shizuka/Shizuka.java

```diff
@@ -0,0 +1,102 @@
+package shizuka;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Shizuka {
+    static final String FILE_PATH = "shizuka.txt";
+
+    public static String[] parseCommand(String args) {
+        return args.split(" ", 2);
+    }
+
+    public static int parseNumber(String args) {
+        int endIndex = args.indexOf(' ') + 1;
+        return Integer.parseInt(args.substring(endIndex));
+    }
+
+    public static void main(String[] args) {
+        Printer.intro();
+        Scanner in = new Scanner(System.in);
+        String line, lineTrimmed;
+        TodoList list0 = new TodoList();
```

Comment:
Consider using the name to explain, as opposed to adding a number.

https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-name-to-explain